plugins/modal/README.md

```diff
@@ -0,0 +1,44 @@
+# @blockly/plugin-modal [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) plugin that creates a modal.
+
+## Installation
+
+### Yarn
+```
+yarn add @blockly/plugin-modal
+```
+
+### npm
+```
+npm install @blockly/plugin-modal --save
+```
+
+## Usage
+
+```js
+import * as Blockly from 'blockly';
+import {Modal} from '@blockly/plugin-modal';
+
+// Inject Blockly.
+const workspace = Blockly.inject('blocklyDiv', {
+  toolbox: toolboxCategories,
+});
+
+// Initialize plugin.
+const modal = new Modal(workspace);
+modal.init();
+```
+
+## API
+- `init`: Create a  modal.
+- `dispose`: Dispose of the modal.
+- `show`: Shows the modal and focus on the first focusable element.
```

Comment:
Shows the modal and focuses on the first interactable element.

Reply:
done